static/src/javascripts/projects/common/modules/commercial/creatives/expandable-v3.js

```diff
@@ -49,80 +51,79 @@ define([
     ExpandableV3.hasScrollEnabled = !detect.isIOS() && !detect.isAndroid();
 
     ExpandableV3.prototype.updateBgPosition = function () {
-        var adHeight, inViewB, inViewT, topCusp, bottomCusp, bottomScroll, topScroll;
-        fastdom.read(function () {
-            adHeight = (this.isClosed) ?
+        var scrollY = window.pageYOffset,
```

Comment:
Tiny remark, you've converted a single `var` definition in one `var`defining 4 variables followed by one `var` per variable.

Not exactly consistent :grin: